modules/timer.py

```diff
@@ -35,8 +35,13 @@ def pidors_callback(bot, job):
     text = 'А вот и список пидарёх: ' + pidors_list_text(job.context['not_pidors'])
     del job.context['pidor_active']
     if len(job.context['not_pidors']) == 1:
-        bot.send_message(chat_id=job.context['chat_id'], text='Sector clear',
+        not_pidor = next(iter(job.context['not_pidors']))
+        bot.send_message(chat_id=job.context['chat_id'],
+                         text='Sector clear. [%s](tg://user?id=%s) единственный не пидор.'
+                              % (all_ids[not_pidor], not_pidor),
```

Comment:
chat_id and text are first positional arguments, so you can omit argument names for them